strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/AccountController.java

```diff
@@ -50,9 +50,9 @@
     private PasswordEncoder passwordEncoder;
     
     @ApiOperation(value = "Get the account details of the currently logged user")
-    @ApiResponses(value = { @ApiResponse(code = 200, message = "Returns account details"),
-                            @ApiResponse(code = 403, message = "Unauthenticated access or user account has been disabled"),
-                            @ApiResponse(code = 404, message = UserController.NOT_FOUND_USER) })
+    @ApiResponses(value = { @ApiResponse(code = org.apache.http.HttpStatus.SC_OK, message = "Returns account details"),
```

Comment:
Could you please replace all `org.apache.http.HttpStatus` with just `HttpStatus`? Thanks!

Reply:
FYI we're using `org.springframework.http.HttpStatus` everywhere. :)

Reply:
Ah, yes! We should use that instead!

Reply:
I used the HttpStatus from Apache because the annotation @ApiResponse requires a code parameter which is an int, it requires a constant and HttpStatus value from Spring is not :)